lib/screens/project_screen.dart

```diff
@@ -0,0 +1,71 @@
+import 'package:flutter/material.dart';
+import 'package:protofast/models/feature.dart';
+import 'package:protofast/models/project.dart';
+import 'package:protofast/screens/srs_screen.dart';
+
+class ProjectScreen extends StatefulWidget {
```

Comment:
How do you think maybe rename it to ProjectFeatures(or just Features) screeen to be clear?